lib/pagseguro/transaction/search.rb

```diff
@@ -86,15 +85,7 @@ def valid?
 
     private
     def perform_request_and_serialize
-      @response = Request.get(@path, api_version,
-      {
-        initialDate: options[:starts_at].xmlschema,
-        finalDate: options[:ends_at].xmlschema,
-        page: page,
-        maxPageResults: options.fetch(:per_page, 50)
-      })
-
-      @errors = Errors.new(@response)
+      raise NotImplementedError.new("'.perform_request_and_serialize' must be implemented in specific search class")
```

Comment:
Yes, I tried. You have to instantiate an object from `NotImplementedError` with a message